src/olympia/addons/tests/test_views.py

```diff
@@ -2453,6 +2606,35 @@ def test_cannot_supply_both_custom_and_license_id(self):
             ]
         }
 
+    def test_source_set_null_clears_field(self):
+        AddonReviewerFlags.objects.create(
+            addon=self.version.addon, needs_admin_code_review=False
```

Comment:
Shouldn't we be setting `needs_admin_code_review` to `True` here? Otherwise, what are we testing at line 2621? If the `update` at line 2613 is setting `needs_admin_code_review` to `True`, then maybe it would be good to add an assert that `self.addon.needs_admin_code_review` is `True` before we do the `patch`.

Reply:
mmm, right.  I'm going to remove the flag clearing code now in any case.